src/main/java/de/uhd/ifi/se/decision/management/jira/webhook/AbstractWebookContentProvider.java

```diff
@@ -0,0 +1,17 @@
+package de.uhd.ifi.se.decision.management.jira.webhook;
+
+import org.apache.commons.httpclient.methods.PostMethod;
+
+
+
+public abstract class AbstractWebookContentProvider  {
+
+
+  String projectKey;
+  WebhookType type;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5269053)